site/src/sphinx/server-grpc.rst

```diff
@@ -200,6 +200,16 @@ entire service implementation in ``RequestContext.current().blockingTaskExecutor
         }
     }
 
+Exception propagation
+======================
+
+It can be very useful to enable ``Flags.verboseResponses()` in your server, which will automatically return
```

Comment:
- ```
  ``Flags.verboseResponses()` -> ``Flags.verboseResponses()``
  ```
- in your server, -> in your server by specifying `-Dcom.linecorp.armeria.verboseResponses=true` JVM option,